modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/breast_milk.dm

```diff
@@ -4,6 +4,7 @@
 	taste_description = "warm and creamy"
 	color = "#ffffffff"
 	reagent_state = LIQUID
+	process_flags = REAGENT_ORGANIC | REAGENT_SYNTHETIC
```

Comment:
Do the non-ERP chemicals even do anything for synthetics? If not there's no point to giving them the process flag

Reply:
yes, they have an arousal stat